src/data/events/highlights.js

```diff
@@ -1,49 +1,37 @@
-const hackschoolDescription = `
-	Come out and learn how to make websites in our beginner-friendly Fall
-	Hackschool series! We will be teaching the basics of web development
-	with React.js, a popular framework of Javascript. No previous experience
-	is necessary!
+const hackSprintDescription = `
+	Come out to ACM Hack's Hack Sprint, a beginner-friendly mobile
+	development workshop series! We'll be covering everything in
+	Swift from building your app from scratch to publishing a fully
+	functional app on the App Store. Our series will end with a c
```

Comment:
Change the word break from `c\nhance` to just `chance` on the next line

Reply:
Again, this was also the formatter. Have no idea why it's doing this.

Reply:
Does the line pass 79 characters with the full word on that line? Formatter is probably trying to stick to convention for line length. You can probably disable that in the rules somewhere, but I'm not aware of how.